pkg/status/status.go

```diff
@@ -374,9 +376,10 @@ func (s *status) monitorClusterStatus() {
 
 			// Report that marketplace is available after meeting minimal syncs.
 			if cohelpers.IsStatusConditionFalse(s.clusterOperator.Status.Conditions, configv1.OperatorAvailable) {
+				reason := "operatorAvailable"
```

Comment:
Same comment here about the version.

Reply:
Same comment as above regarding adding version to the reason.

proposed `Reason` if we add the version:
`OperatorProgressedTo4.2.0-0.ci-2019-07-10-180724`.